src/utils/responseParser/sequencer.ts

```diff
@@ -211,4 +212,12 @@ export class SequencerAPIResponseParser extends ResponseParser {
       },
     };
   }
+
+  // TODO: Define response as new type as it diff from ContractClass
+  public parseSierraContractClassResponse(res: any): SierraContractClass {
```

Comment:
Nitpick: Not sure if it would make sense to put this somewhere else since `abi` property of the RPC response will also be a string.

Reply:
At the moment until protocol stabilized both seq. and rpc. had their own response parser.